src/sharp_plugin.c

```diff
@@ -171,6 +171,10 @@ ncclResult_t ncclSharpInit(ncclDebugLogger_t logFunction) {
   gettimeofday(&tval, NULL);
   srand((int) tval.tv_usec);
 
+  /* set SHARP COLL library default for plugin */
+  setenv("SHARP_COLL_ENABLE_SAT", "1", 0);
```

Comment:
Is it possible to set this in sharp_coll_init_spec? If left flag here it may affect hcoll behavior if mpi_init is called after ncclInit

Reply:
currently, we do not have  coll_init_spec flag to do this. dont want to break ABI for this.  will add in future changes. 
HCOLL/SHARP is enabled only with HCOLL_ENABLE_SHARP. I think users won't do this in NCCL use cases.

Reply:
Maybe just for future sharp releases it would be good to have config_read, config_modify like in ucx.